src/power_grid_model/data_types.py

```diff
@@ -15,28 +15,30 @@
 
 SparseBatchArray = Dict[str, np.ndarray]
 """
-A sparse batch array is a dictionary containing the keys "indptr" and "data".
-    indptr: a one-dimensional numpy int32 array
-    data: a one-dimensional structured numpy array. The exact dtype depends on the type component.
+A sparse batch array is a dictionary containing the keys `indptr` and `data`.
 
-Example: {"indptr": <1d-array>, "data": <1d-array>}
+- indptr: a one-dimensional numpy int32 array
```

Comment:
@mgovers @zhen0427 this is actually not correct. The `indptr` uses `int64` integers. Please change this in all the places.

Reply:
Fixed in https://github.com/PowerGridModel/power-grid-model/pull/518